src/atlas4py/CMakeLists.txt

```diff
@@ -57,14 +59,16 @@ if(NOT atlas_FOUND)
 
     if(_atlas4py_built_eckit)
         # copy atlas' eckit dependencies
-        get_target_property(deps atlas LINK_LIBRARIES)
-        get_target_property(deps_interface atlas INTERFACE_LINK_LIBRARIES)
-        list(APPEND deps ${deps_interface})
-        list(REMOVE_DUPLICATES deps)
-        foreach(dep IN LISTS deps)
-            if(dep MATCHES "eckit.*")
-                copy_target(${dep})
-            endif()
+        foreach(lib "atlas" "atlas_io")
+            get_target_property(deps ${lib} LINK_LIBRARIES)
+            get_target_property(deps_interface ${lib} INTERFACE_LINK_LIBRARIES)
+            list(APPEND deps ${deps_interface})
+            list(REMOVE_DUPLICATES deps)
+            foreach(dep IN LISTS deps)
+                if(dep MATCHES "eckit.*")
+                    copy_target(${dep})
+                endif()
```

Comment:
I think it's more consistent if we build the full list here and deduplicate. Now we are mixing patterns...